packages/devextreme/scss/widgets/fluent/form/_index.scss

```diff
@@ -41,14 +41,18 @@
 .dx-layout-manager {
   .dx-field-item {
     &:not(.dx-last-col) {
-      padding-right: 20px;
+      padding-right: 8px;
```

Comment:
Let's use scss variables.

Reply:
Extract it please in variable as follows:
`
$form-layout-manager-padding: 8px;
`